src/controllers/home-page.controller.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {get} from '@loopback/openapi-v3';
+
+const pkg = require('../../../package.json');
+
+export class HomePageController {
+  name: string;
+  version: string;
+  description: string;
+
+  constructor() {
+    this.name = pkg.name;
+    this.version = pkg.version;
+    this.description = pkg.description || pkg.name;
+  }
+
+  @get('/', {
+    responses: {
+      '200': {
+        description: 'Home Page',
+        content: {'text/html': {schema: {type: 'string'}}},
+      },
+    },
+  })
+  homePage() {
+    return `
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>${this.description}</title>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
```

Comment:
Can we skip out on loading the full `bootstrap.min.css`  and the accompanying `js` file? We're using about 2 styles from this entire file. I'd rather we copy the style in-lined for a faster render. No JS is needed. (Currently this + JS is loading 63.1 KB while the page payload is only 1.7 KB). 

An alternative IIRC is that BootStrap also provides just the `grid` CSS -- which is lighter and includes the CSS for `container` class.